internal/engine/actions/alert/pull_request_comment/pull_request_comment.go

```diff
@@ -245,6 +190,28 @@ func (_ *Alert) runDoNothing(ctx context.Context, params *paramsPR) (json.RawMes
 	return nil, err
 }
 
+func (alert *Alert) runDoReview(ctx context.Context, params *paramsPR) (json.RawMessage, error) {
+	sac := actionContext.GetSharedActionsContext(ctx)
+	if sac == nil {
+		return nil, fmt.Errorf("shared actions context not found")
+	}
+
+	// This was a successful result, so we don't need to alert
+	if !params.shouldAlert {
```

Comment:
Why do you need to check for this here? My understanding is the `shouldAlert` flag is used to tell what the alert command should be.